Packs/VMRay/Integrations/VMRay/VMRay.py

```diff
@@ -168,49 +162,31 @@ def find_error(may_be_error_inside):
                     return err_r
         return None
 
-    def reset_file_buffer(files):
-        """ The requests library reads the file buffer to the end.
-            If we want to try to submit again, we need to set the file buffer to
-            beginning manually.
-        Args:
-            files: (dict)
-        """
-
-        if not isinstance(files, dict):
-            return
-
-        try:
-            fobj = files["sample_file"][1]
-            fobj.seek(0, 0)
-        except (IndexError, KeyError):
-            return
```

Comment:
Hochmand added it when writing the integration, since then we implemented BaseClient and we are doing the same calls (with file) using the BaseClient without reseting the buffer.